setup.py

```diff
@@ -108,7 +108,7 @@ def cpp_flag(compiler):
 class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
 
-    c_opts = {"msvc": ["/EHsc", "/bigobj"], "unix": []}
+    c_opts = {"msvc": ["/EHsc", "/bigobj"], "unix": ["-g0"]}
```

Comment:
I didn't know that you can set that flag. It is not documented in Apple Clang.

Reply:
Clang tends to allow GCC opts. This fixed the issue on manylinux, and macOS was already smaller.